osu.Game/Screens/Edit/Verify/VisibilitySection.cs

```diff
@@ -28,8 +35,18 @@ private void load(OverlayColourProvider colours)
                     LabelText = issueType.ToString()
                 };
 
-                checkbox.Current.BindTo(verify.ShowIssueType[issueType]);
-                checkbox.Current.BindValueChanged(_ => verify.IssueList.Refresh());
+                checkbox.Current.Default = !verify.HiddenIssueTypes.Contains(issueType);
```

Comment:
You'll also likely want to create a local bound copy to avoid cross-component bindings:

```csharp
hiddenIssueTypes = verify.HiddenIssueTypes.GetBoundCopy();
```

and use that field in this class rather than direct references to `verify.`.

Reply:
Out of curiousness, are cross-component bindings are mainly an issue due to the bindings becoming unorganized and more difficult to debug/trace, or am I missing the point?

Reply:
To give a very brief history lesson, `Bindable`s are our solution to event/memory leaks caused by using the classic `event` binding pattern (ie. `ThingChanged += ourThing` and forgetting to remove the binding). To make this work properly, each class needs to maintain their own instance.

If this rule is broken, as an example, your `VisibilitySection` is holding a reference to its parent (because you are binding a local method with implicit references to fields of the instance). While generally this isn't an issue because you presume the parent is cleaned up along with its children, it is a larger issue in the opposite direction. Imagine doing something like `game.CurrentScreen.BindValueChanged(screen => { this.currentScreenText.Text = $"{screen}"; })`. Your local instance would now be held for the full duration of game execution.

For the sake of good practices we tend to avoid this in both directions.

There's one more reason: `Drawable` has code to automatically unbind all bindable fields/properties when an instance goes out of context. This stops events from firing at all, where firing them could lead to unexpected behaviour, performance implications, or cross-thread exceptions.

Reply:
That makes sense. I've applied the same change to `InterpretationSection`, as I figure the same thing applies there (and in doing so I also realized I forgot to bind it to refresh on update, so did that in `IssueList` like with `HiddenIssueTypes`).